### PR TITLE
fix(chunjun-connector-hive) [chunjun-1350]

### PR DESCRIPTION
**Pull request by ironZr**
## Purpose of this pull request

修复了没有读取配置的index而直接for循环用当前元素下标做index

## Which issue you fix
Fixes # (issue).
#1350

## Checklist:

- [x] I have executed the **'mvn spotless:apply'** command to format my code.
- [x] I have a meaningful commit message (including the issue id, **the template of commit message is '[label-type-#issue-id][fixed-module] a meaningful commit message.'**)
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have checked my code and corrected any misspellings.
- [x] My commit is only one. (If there are multiple commits, you can use 'git squash' to compress multiple commits into one.)

### CODE REVIEW

File: chunjun-connectors/chunjun-connector-hive/src/main/java/com/dtstack/chunjun/connector/hive/util/HiveUtil.java [hunk 1]
```diff
@@ -47,13 +47,7 @@
...
 import java.sql.ResultSet;
 import java.sql.Timestamp;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.TreeMap;
+import java.util.*;
```
Comment (FlechazoW) [inline chunjun-connectors/chunjun-connector-hive/src/main/java/com/dtstack/chunjun/connector/hive/util/HiveUtil.java:50]: Do not import *.

**Comment by FlechazoW** [comment]
You need to execute './mvnw spotless:apply' to format the code.

**Comment by ironZr** [comment]
> You need to execute './mvnw spotless:apply' to format the code.

好的 谢谢提醒，以按要求修改

**Comment by FlechazoW** [comment]
DNM.

**Pull request closed without merging**